x/staking/types/validator.go

```diff
@@ -477,27 +474,6 @@ func (v Validator) ConsPubKey() (cryptotypes.PubKey, error) {
 	return pk, nil
 }
 
-// CmtConsPublicKey casts Validator.ConsensusPubkey to cmtprotocrypto.PubKey.
-func (v Validator) CmtConsPublicKey() (cmtprotocrypto.PublicKey, error) {
```

Comment:
Should the methods be deprecated instead to avoid breaking changes?

Reply:
probably, its always nicer to remove things so its easy for us